Embedded Metadata.js

```diff
@@ -1256,7 +1260,7 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"date": "2013/02/20",
+				"date": "2013/2/20",
```

Comment:
Unrelated, but seems like we should be cleaning these…

Reply:
I'm pretty sure the only thing we'd have is a `ZU.strToISO` in there, which would be fine (I think?) if duplicated.